project/fr-FR/Mod/No Pollution.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Options.SECTION[NoPollution.NoPollution.Mod]": "No Pollution",
+  "Options.GROUP[NoPollution.NoPollution.Mod.Resetting]": "Réinitialiser la pollution sonore",
+  "Options.OPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollutionReset]": "Pollution sonore",
+  "Options.OPTION_DESCRIPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollutionReset]": "Ce bouton devrait réinitialiser la pollution sonore.",
+  "Options.GROUP[NoPollution.NoPollution.Mod.PollutionToggles]": "Activer/désactiver les types de pollution",
+  "Options.OPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollution]": "Pollution sonore",
+  "Options.OPTION_DESCRIPTION[NoPollution.NoPollution.Mod.ModSettings.NoisePollution]": "Autoriser ou non la pollution sonore (coché signifie que la pollution est active)",
```

Comment:
I'd replace *"Autoriser ou non la pollution [...] (coché signifie que la pollution est active)"* by *"Activer l’émission de pollution par [...]."*
I know this is not exactly the phrasing of the mod, but the original phrasing is bad and generally checkboxes have a much simpler description, like I proposed.